gensrc/thrift/FrontendService.thrift

```diff
@@ -604,6 +604,12 @@ struct TSnapshotLoaderReportRequest {
     5: optional i32 total_num
 }
 
+// (start_rowset_id, end_rowset_id) not include start and end
+struct TGetRowsetIdResponse {
+    1: optional i64 start_rowset_id = -1
+    2: optional i64 end_rowset_id = -1
```

Comment:
I prefer [start, end) with start_id and id_count

Reply:
done